ledger-core/virtual/integration/delegationtoken_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package integration
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gojuno/minimock/v3"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/insolar/assured-ledger/ledger-core/insolar/contract"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/jet"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/node"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/runner/logicless"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/authentication"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/integration/utils"
+)
+
+func TestDelegationToken_CheckTokenField(t *testing.T) {
+	t.Skip("https://insolar.atlassian.net/browse/PLAT-588")
+	tests := []struct {
+		name         string
+		testRailID   string
+		fakeCaller   bool
+		fakeCallee   bool
+		fakeOutgoing bool
+	}{
+		{
+			name:         "Fail with wrong caller in token",
+			testRailID:   "C5197",
+			fakeCaller:   true,
+			fakeCallee:   false,
+			fakeOutgoing: false,
+		},
+		{
+			name:         "Fail with wrong callee in token",
+			testRailID:   "C5198",
+			fakeCaller:   false,
+			fakeCallee:   true,
+			fakeOutgoing: false,
+		},
+		{
+			name:         "Fail with wrong outgoing in token",
+			testRailID:   "C5198",
+			fakeCaller:   false,
+			fakeCallee:   false,
+			fakeOutgoing: true,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			t.Log("C5197")
+
+			mc := minimock.NewController(t)
+
+			server, ctx := utils.NewUninitializedServer(nil, t)
+			defer server.Stop()
+
+			runnerMock := logicless.NewServiceMock(ctx, mc, nil)
+			server.ReplaceRunner(runnerMock)
```

Comment:
I'm not sure, that you need to replace runner.

Reply:
fix